mne_bids/tests/test_read.py

```diff
@@ -120,16 +120,19 @@ def test_read_participants_data(tmp_path):
     subject_info = {
         'hand': 1,
         'sex': 2,
+        'weight': 70.5,
+        'height': 180.5
```

Comment:
can you assert somewhere that the values of 70.5 and 180.5 are set properly?

Reply:
This was previously not possible as we'd only **read** weight and height from an existing participants.tsv, but never **write** to it.

I have now added this functionality.